### PR TITLE
[Refactor] Updated Dashboard Component

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: YashKamboj <yashkamboj29@gmail.com>

**Description**
Added serivice mesh table in the dashboard page 

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by YashKamboj** [comment]
This PR is ready to merge

File: ui/components/Dashboard/showServiceMesh.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+     function getSelectedK8sContextsNames ()  {
+        return getK8sClusterNamesFromCtxId()
+      } 
+    
+      function Meshcard (mesh, components = []) {
+  
+        if (Array.isArray(components) && components.length)
+          return (
+            <Paper elevation={1} style={{ padding : "2rem", marginTop : "1rem" }}>
```
Comment (Chadha93) [inline ui/components/Dashboard/showServiceMesh.js:27]: inline styling.... not using sx props? @YashKamboj

File: ui/components/Dashboard/showServiceMesh.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {components
+                      .filter((comp) => comp.namespace === activeMeshScanNamespace[mesh.name])
+                      .map((component) => {
+                        return (
+                          <TableRow key={component.name.full}>
+                            <TableCell align="center">{component.component}</TableCell>
+                            {/* <TableCell align="center">{versionMapper(component.version)}</TableCell> */}
```
Comment (Chadha93) [inline ui/components/Dashboard/showServiceMesh.js:62]: remove the unwanted code line.

File: ui/components/Dashboard/showServiceMesh.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+                            <Tooltip
+                              key={`component-${component.name}`}
+                              title={
+                                Array.isArray(component?.data_planes) && component.data_planes.length > 0 ? (
+                                  component.data_planes.map((cont) => {
+                                    return (
+                                      <div key={cont.name} style={{ fontSize : "15px", color : '#fff', paddingBottom : '10px', padding : '1vh' }}>
```
Comment (Chadha93) [inline ui/components/Dashboard/showServiceMesh.js:69]: here too

File: ui/components/Dashboard/showServiceMesh.js [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+            >
+                <Typography style={{ fontSize : "1.5rem", marginBottom : "2rem" }} align="center" color="textSecondary">
+                  {emptyStateMessageForServiceMeshesInfo}
+                </Typography>
+                <Button
+                  aria-label="Add Meshes"
+                  variant="contained"
+                  color="primary"
+                  size="large"
+                //   onClick={() => self.props.router.push("/management")}
```
Comment (Chadha93) [inline ui/components/Dashboard/showServiceMesh.js:163]: No event for this button??

**Comment by Abhishek-kumar09** [comment]
@YashKamboj Would you post a screenshot

**Pull request closed without merging**